pkg/menu/menu.go

```diff
@@ -1,15 +1,28 @@
 package menu
 
 import (
+	"fmt"
 	"io"
+	"strconv"
+	"strings"
 
 	ui "github.com/gizak/termui/v3"
 	"github.com/gizak/termui/v3/widgets"
 )
 
 const menuWidth = 50
 
-// newParagraph will return a paragraph object with given initial text.
+// Entry contains all the information needed for a boot entry.
+type Entry interface {
+	// Label is the string will show in menu.
+	Label() string
+	// IsDefault returns true means this entry is default.
```

Comment:
Nit: I think this is self explanatory, maybe remove this comment?

Reply:
golint ask me to have a comment to explain the IsDefault